src/components/atoms/TabMenu/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from "react";
```

Comment:
A lot of this is duplicate from the TabArea atom. Is there any way you can modify TabArea to meet your needs?

Reply:
At first I applied all modifications on TabArea , but since we needed here in this feature to show action section on top I thought that all modification will make the tabArea very complex and also TabArea has this icon on top header so I though we needed this new component to make the code easier and to have this action section on top

Reply:
As agreed we will rename it to be TabSection and keep the new component